pages/linux/lxc.md

```diff
@@ -0,0 +1,31 @@
+# lxc
+
+> Manages Linux containers (using the lxd REST API).
+
+- List all containers:
+
+`lxc list [{{remote}}:][{{filter}}]`
```

Comment:
I'm not sure I understand. Which of the possible combinations of these parameters are valid? Can one use either, both, or only one or the other? Or does one of them require the other, but not the other way around?